button_views/leaderboard_buttons.py

```diff
@@ -42,7 +42,7 @@ async def go_to_page_one(
         await interaction.response.defer(ephemeral=False)
 
     @discord.ui.button(
-        label="Previous", row=0, style=discord.ButtonStyle.primary, emoji="⬅️"
+        label="UP", row=0, style=discord.ButtonStyle.primary, emoji="⬅️"
```

Comment:
Up and down are contradicting to "⬅️" and "➡️"

Reply:
How do I format the Emojis, so Discord can process them?

Reply:
Just use UFT-8 Emojis. Otherwise, read [discord.ui.button doc](https://discordpy.readthedocs.io/en/stable/interactions/api.html?highlight=discord%20ui%20button#discord.ui.button).

Reply:
In Germany we say: ich versteh nur Bahnhof
Unicode doesnt work
Pasting Emoji doenst work
Output reads invalid

Reply:
Just copy the button emoji from discord and insert it as a string, link this: "⏫".
The emoji must be also be available as a Unicode character ⏫ (https://apps.timwhitlock.info/unicode/inspect/hex/23EB).
If you want to use a discord only emoji, you need to use a [PartialEmoji](https://discordpy.readthedocs.io/en/stable/api.html#discord.PartialEmoji) or [Emoji](https://discordpy.readthedocs.io/en/stable/api.html#discord.Emoji). PartialEmojis can be constructed from a string as seen [here](https://discordpy.readthedocs.io/en/stable/api.html#discord.PartialEmoji.from_str).

Reply:
Did it... somehow

Reply:
> Just copy the button emoji from discord and insert it as a string, link this: "⏫". The emoji must be also be available as a Unicode character ⏫ (https://apps.timwhitlock.info/unicode/inspect/hex/23EB).

Found same website. now it works